sdk/typescript/package.json

```diff
@@ -32,7 +32,7 @@
   "scripts": {
     "build": "tsc",
     "watch": "tsc -w",
-    "test": "yarn run test:node && yarn run test:bun",
+    "test": "yarn run test:node",
```

Comment:
`test` should *just* run the old node tests, but functionally, I don't think this is actually run anywhere from the codebase so just provides some backwards compat.